rladies.qmd

```diff
@@ -0,0 +1,20 @@
+---
+title: "R-Ladies Sheffield"
+image: images/rladies_hex.png
+about:
+  template: jolla
+  links:
+    - icon: twitter-x
+      text: Twitter
+      href: https://twitter.com/RLadiesSheff
+---
+
+::: {style="text-align: left;"}
+
+We are Sheffield R-ladies, a chapter of [R-Ladies Global](https://rladies.org/). The mission of R-Ladies is to achieve proportionate representation by encouraging, inspiring, and empowering people of genders currently under-represented in the R community. R-Ladies’ primary focus, therefore, is on supporting minority gender R enthusiasts to achieve their programming potential, by building a collaborative global network of R leaders, mentors, learners, and developers to facilitate individual and collective progress worldwide.
```

Comment:
This and the following paragraph are failing the [markdownlint-cli2](https://github.com/DavidAnson/markdownlint/) checks performed by [pre-commit](https://pre-commit.com).

Lines need hard-wrapping at 120 characters (from memory RStudio soft-wraps lines).